drivers/mtd/nand/raw/gpmi-nand/gpmi-nand.c

```diff
@@ -77,7 +77,7 @@ static int clear_poll_bit(void __iomem *addr, u32 mask)
 static int gpmi_reset_block(void __iomem *reset_addr, bool just_enable)
 {
 	int ret;
-	int timeout = 0x400;
+	int timeout = 0x600; /* Increase from 400 for PORTF-1119 */
```

Comment:
Increase from 0x400 that is.

Might be interesting to know the median/average value of `timeout` at the end of this routine in the case of successful return.